common/test/acceptance/tests/lms/test_progress_page.py

```diff
@@ -313,6 +313,7 @@ def _check_tick_text(self, index, sr_text, label, label_hidden=True):
         self.assertEqual(sr_text, self.progress_page.x_tick_sr_text(index))
         self.assertEqual([label, 'true' if label_hidden else None], self.progress_page.x_tick_label(index))
 
+    @attr('a11y')
```

Comment:
This is failing the a11y test and is unrelated to these changes, so maybe just revert it...